backend/database.js

```diff
@@ -36,32 +37,49 @@ if (!fs.existsSync(dataFolder)) {
   fs.mkdirSync(dataFolder, { mode: 0o750 })
 }
 
+// Streams stored contract log entries since the given entry hash (inclusive!), most recent first.
 sbp('sbp/selectors/register', {
   'backend/db/streamEntriesAfter': async function (contractID: string, hash: string): Promise<*> {
     const latestHEADinfo = await sbp('chelonia/db/latestHEADinfo', contractID)
     if (!latestHEADinfo) {
       throw Boom.notFound(`contractID ${contractID} doesn't exist!`)
     }
-    let { HEAD: currentHEAD } = latestHEADinfo
+    const entries = []
+    // Number of entries pushed.
+    let counter = 0
+    let currentHash = hash
     let prefix = '['
+    try {
+      while (currentHash && (entries.length < MAX_EVENTS_AFTER)) {
+        const entry = await sbp('chelonia/db/getEntry', currentHash)
+        if (entry) {
+          entries.push(entry)
+          currentHash = await sbp('chelonia/db/get', `next=${currentHash}`)
+        }
+      }
+    } catch (e) {
+      console.error(`read(): ${e.message}:`, e)
+    }
+    entries.reverse()
```

Comment:
The entries were reversed before because they had to be (since we couldn't go forwards).

Now that that's no longer the case, there's no reason to artificially reverse the entries, so please remove this line and update the other code accordingly.

I've gone ahead and also opened an issue to update the `chel` command once this PR is merged: https://github.com/okTurtles/chel/issues/33